Warehouse.Api.Auth/Business/AuthService.cs

```diff
@@ -43,46 +47,51 @@ public async Task<Result<UserDto>> RegisterAsync(RegisterDto register)
             var user = Mapper.Map<UserDto>(register);
             var hashedPassword = _hasher.HashPassword(user, register.Password);
             user = user with {PasswordHash = hashedPassword, Roles = new List<string> {"User"}};
-            await _userService.CreateAsync(user);
+            await _sender.SendMessage(user, "CreateUserQueue");
             return Result<UserDto>.Success(user);
         }
 
         public async Task<Result<UserAuthenticatedDto>> LoginAsync(LoginDto login, string sessionId)
         {
-            var user = (await _userService.GetByUserNameAsync(login.UserName)).Data with {SessionId = sessionId};
-            IsValid(user, login.Password);
+            var user = await _userRepository.GetByUserNameAsync(login.UserName);
+            user.SessionId = sessionId;
+            var userDto = Mapper.Map<UserDto>(user);
 
-            await _userService.UpdateAsync(user);
+            IsValid(userDto, login.Password);
 
-            var jwtToken = await GenerateJwtToken(user);
+            await _sender.SendMessage(user, "UpdateUserQueue");
+
+            var jwtToken = await GenerateJwtToken(userDto);
             var tokenString = await GenerateRefreshToken();
```

Comment:
GenerateJwtToken and GenerateRefreshToken are synchronous methods made asynchronous with Task.FromResult. Do we really need them to be asynchronous?